packages/terra-search-field/tests/jest/SearchField.test.jsx

```diff
@@ -221,7 +225,7 @@ describe('Auto Search', () => {
     searchField.find('.input').simulate('change', { target: {} });
 
     for (let i = 0; i < 3; i += 1) {
-      jest.advanceTimersByTime(250);
+      jest.advanceTimersByTime(2500);
       expect(onSearch).not.toBeCalled();
     }
```

Comment:
Loops are not recommended in unit tests. Since this is executed 3 times, can we break it out?